automated/linux/usb/test-usb-2.yaml

```diff
@@ -0,0 +1,19 @@
+metadata:
+    name: Test usb-2
+    format: "Lava-Test Test Definition 1.0"
+    description: "Check MD5."
```

Comment:
Please change description. We don't want to test the md5 tool, we want to check, if USB transfer works correct by copying a file onto a flash disk and compare its md5sum with the one from the source file. Something like "Test for correct USB data transfer with a USB flash disk."